code/modules/mob/living/living_combat.dm

```diff
@@ -0,0 +1,88 @@
+/mob/living/ComponentInitialize()
+	. = ..()
+	RegisterSignal(src, SIGNAL_TRAIT(TRAIT_COMBAT_MODE_LOCKED), .proc/update_combat_lock)
+
+/mob/living/proc/update_combat_lock()
+	var/locked = IS_COMBAT_MODE_LOCKED(src)
+	var/desired = IS_COMBAT_TOGGLED(src)
+	var/actual = IS_COMBAT_ACTIVE(src)
+	if(actual)
+		if(locked)
+			disable_combat_mode(FALSE, TRUE, FALSE, FALSE)
+		else if(!desired)
+			disable_combat_mode(TRUE, TRUE, FALSE, FALSE)
+	else
+		if(desired && !locked)
+			enable_combat_mode(FALSE, TRUE, FALSE, FALSE)
+	update_combat_mode_icon()
+
+/mob/living/proc/disable_combat_mode(silent = TRUE, was_forced = FALSE, visible = FALSE, update_icon = TRUE)
+	if(!IS_COMBAT_ACTIVE(src))
+		return
+	DISABLE_BITFIELD(combat_flags, COMBAT_FLAG_COMBAT_ACTIVE)
+	SEND_SIGNAL(src, COMSIG_LIVING_COMBAT_DISABLED, was_forced)
+	if(visible)
+		visible_message("<span class='warning'>[src] goes limp.</span>", "<span class='warning'>Your muscles are forceibly relaxed!</span>")
+	else if(!silent)
+		to_chat(src, was_forced? "<span class='warning'>Your muscles are forceibly relaxed!</span>" : "<span class='warning'>You relax your muscles.</span>")
+	if(update_icon)
+		update_combat_mode_icon()
+
+/mob/living/proc/enable_combat_mode(silent = TRUE, was_forced = FALSE, visible = FALSE, update_icon = TRUE)
+	if(IS_COMBAT_ACTIVE(src))
+		return
+	ENABLE_BITFIELD(combat_flags, COMBAT_FLAG_COMBAT_ACTIVE)
+	SEND_SIGNAL(src, COMSIG_LIVING_COMBAT_ENABLED, was_forced)
+	if(visible)
+		visible_message("<span class='warning'>[src] drops into a combative stance!</span>", "<span class='warning'>You drop into a combative stance!</span>")
+	else if(!silent)
+		to_chat(src, was_forced? "<span class='warning'>Your muscles reflexively tignten!</span>" : "<span class='warning'>You tense your muscles.</span>")
```

Comment:
tighten

Reply:
i was actually referring to the typo here, not to use a different word lol. "tignten"